docs/hooks/index.md

```diff
@@ -82,52 +82,59 @@ Only the `encounterId` field in this example is eligible to be a prefetch token
     "gender": "male",
     "birthDate": "2024-08-12"
   }
+}
 ```
 
 ## Hook Definition Format
 
-Hooks are defined in the following format:
-
------
+Hooks are defined in the following format.
 
-# `hook-name-expressed-as-noun-verb` 
+### `hook-name-expressed-as-noun-verb`
 
-The name of the hook SHOULD succinctly and clearly describes the activity or event. Hook names are unique so hook creators SHOULD take care to ensure newly proposed hooks do not conflict with an existing hook name. Hook creators MAY choose to name their hook with a URI (e.g. `https://example.org/hooks/patient-transmogrify`) if the hook is specific to an organization.
+The name of the hook SHOULD succinctly and clearly describe the activity or event. Hook names are unique so hook creators SHOULD take care to ensure newly proposed hooks do not conflict with an existing hook name. Hook creators MAY choose to name their hook with a URI (e.g. `https://example.org/hooks/patient-transmogrify`) if the hook is specific to an organization.
 
 When naming hooks, the name should start with the subject (noun) of the hook and be followed by the activity (verb). For example, `patient-view` (not `view-patient`) or `medication-prescribe` (not `prescribe-medication`).
 
-## Workflow
+### Workflow
 
 Describe when this hook occurs in a workflow. Hook creators SHOULD include as much detail and clarity as possible to minimize any ambiguity or confusion amongst implementors.
 
-## Context
+### Context
 
 Describe the set of contextual data used by this hook. Only data logically and necessarily associated with the purpose of this hook should be represented in context.
 
-All fields defined by the hook's context MUST be defined in a table where each field is described by three attributes:
+All fields defined by the hook's context MUST be defined in a table where each field is described by the following attributes:
+
+- Field: The name of the field in the context JSON object.
+- Optionality: A string value of either `REQUIRED` or `OPTIONAL`
+- Prefetch Token: A string value of either `Yes` or `No`, indicating whether this field can be tokenized in a prefetch template.
+- Type: The type of the field in the context JSON object, expressed as the JSON type, or the name of a FHIR Resource type. Valid types are *string*, *number*, *object*, *array*, or the name of a FHIR resource type. When a field can be of multiple types, type names MUST be separated by a *pipe* (`|`)
```

Comment:
What about types `true`, `false`, and `null` (all of which are allowed by JSON)?

Reply:
Agreed, I added *boolean* to the list. Null is allowed for the values, but I don't think we'd want the type not to be specified, right?

Reply:
Oh, good point! Yes, null shouldn't be a type 😄